resources/user-agents/browsers/nokia-browser/nokia-browser-7-1.json

```diff
@@ -29,23 +29,35 @@
       },
```

Comment:
For the most part, I just condensed devices for the version Nokia files. The majority of the device adds were in the unversioned "nokia-browser.json" file.